Extensions/xkit_patches.js

```diff
@@ -1344,6 +1575,13 @@ XKit.tools.getParameterByName = function(name){
 
 			},
 
+			/**
+			 * Instantiate and add a previously "created" button to the
+			 * specified post.
+			 * @param {Object} obj - Interface Post Object
+			 * @param {String} class_name - CSS class of the button to be added
+			 * @param {String?} additional - Additional parameters for the button's div
```

Comment:
unclear. Should mention that this is in the form of a string that get inserted into the button's tag.